storage/EtcFstab.cc

```diff
@@ -690,16 +690,29 @@ namespace storage
 	{
 	    string path = mount_entry->get_mount_point();
 
+	    // for multiple mount points for same partition try at first identical path
+	    // and if not found, then any
 	    vector<JointEntry>::iterator it = find_if(ret.begin(), ret.end(), [&path](const JointEntry& tmp) {
                 bool in_fstab = tmp.is_in_etc_fstab();
                 if (in_fstab && tmp.fstab_entry->get_mount_point() != path)
                 {
-                    y2war("mount points for " << tmp.fstab_entry->get_device()
-                          << " differ: fstab(" << tmp.fstab_entry->get_mount_point()
-                          << ") != proc(" << path << ")");
+		    return false;
```

Comment:
NP: It was a little bit difficult to understand this to me. Why not a more explicit comparison?

```
if (!in_fstab)
  return false;

tmp.fstab_entry->get_mount_point() == path
```